tests/system/ak_test.go

```diff
@@ -21,42 +21,72 @@ import (
 	"context"
 	"fmt"
 	"io/fs"
-	"path/filepath"
 	"strings"
 	"testing"
 	"time"
+
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
 )
 
 const (
-	rootDir     = "testdata/"
+	rootDir     = "testdata"
```

Comment:
I know you change just removed "/" (why?), but this raises a bigger question - should we rename this const? It's confusing it see it here, but also as a param in `fs.WalkDir`.

Reply:
looks like it needs to be without the `/` for WalkDir but with the `/` for TrimPrefix.